packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -20,17 +18,20 @@ import {
   stopHost,
 } from 'xo'
 import {
+  AssertConsistentHostServerTime,
   connectStore,
   formatSizeShort,
   hasLicenseRestrictions,
   osFamily,
 } from 'utils'
+import { Col, Row } from 'grid'
```

Comment:
Not related change.

Reply:
I made some changes that require  `Sort`.

Reply:
It's  not sorted.

Reply:
Inline imports before multiple lines imports.